added the show commands in the docs (#2951)
hashicorp/web-unified-docs: content/terraform/v1.16.x (beta)/docs/cli/commands/stacks/deployment-run/show.mdx

---
page_title: terraform stacks deployment-run show command
description: The terraform stacks deployment-run show command shows details of a deployment run.
# START AUTO GENERATED METADATA, DO NOT EDIT
created_at: 2026-08-03T12:10:21-05:00
last_modified: 2026-08-13T20:21:48.000Z
# END AUTO GENERATED METADATA
---

# `terraform stacks deployment-run show` reference

Use the `terraform stacks deployment-run show` command to show details of a deployment run.

## Usage

```shell-session
$ terraform stacks <global-stacks-flags> deployment-run show <options>
```

## Description

The `terraform stacks deployment-run show` command retrieves relevant information for a given deployment run, including its deployment steps.

> You can identify the target deployment run either by providing a deployment run ID directly.

Reviewer: ```suggestion
You can target a deployment run by providing its ID.
```

## Options

- `-deployment-run-id`: The ID of the deployment run to show.
    - Required.
    - String data type.

> - `-json`: Output results in JSON format.
>     - Optional.
>     - Boolean flag.
>     - Default is false (human-readable output).

Reviewer: ```suggestion
- `-json`: Formats the output in JSON format. When set to `false`, Terraform formats the output as human-readable text.
    - Optional.
    - Boolean flag.
    - Default is `false`.
```
